packages/browser-utils/src/parse-html.ts

```diff
@@ -12,10 +12,13 @@ export function parseHTML(document: Document, html: string | HtmlSafeString): Do
  * {@link DocumentFragment}. If the HTML being parsed does not contain
  * exactly one element, an error is thrown.
  */
-export function parseHTMLElement(document: Document, html: string | HtmlSafeString): Element {
+export function parseHTMLElement<T extends Element>(
```

Comment:
You probably want a default to avoid this becoming a breaking change:

```suggestion
export function parseHTMLElement<T extends Element = Element>(
```